scripts/azureml-assets/azureml/assets/environment/pin_package_versions.py

```diff
@@ -23,6 +23,8 @@
 
 def create_package_finder(index_urls: List[str]) -> PackageFinder:
     """Create a pip PackageFinder."""
+    print("List of index urls: " + str(index_urls))
+    logger.log_debug("List of index urls: " + str(index_urls))

```

Comment:
Please revert the changes to this file after you're done testing/debugging.

Reply:
done! reverted the changes to the file.